src/i18n/currencies.ts

```diff
@@ -0,0 +1,8 @@
+/**
```

Comment:
I think we can remove this comment as @tajespasarela mentioned before, it makes no sense.